theme/templates/base.html

```diff
@@ -39,75 +38,70 @@ <h1><a href="{{ SITEURL }}/">{{ SITENAME }} {% if SITESUBTITLE %} <strong>{{ SIT
                 {% endif %}
                 </ul></nav>
         </header><!-- /#banner -->
-        <div class="content-wrapper">
-            <div class="sidebar">
-                <h2>Conferences</h2>
-                {% if CONFERENCE_LINKS %}
+
+        <div class="content-wrapper" style="position:relative;">
```

Comment:
Looks like another indentation issue here, let's make sure we are consistent between tabs and spaces everywhere.  Thanks!